### PR TITLE
Add a new PS script displaying test cases

### PR DESCRIPTION
**Pull request by juhlee-microsoft**
list out test cases
show the counts of test cases per platform
tag information

### CODE REVIEW

File: Get-LISAv2Statistics.ps1 [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+$tags = @{}
+
+Write-Output ""
+“{0,-60} {1,20} {2,15} {3,15} {4,40}” -f` "TestCase", "Platform", "Category", "Area", "Tags"
+Write-Output "----------------------------------------------------------------------------------------------------------------------------------------------------------"
+
+foreach ($fname in $files) {
+    $xml = [xml](Get-Content $fname)
+    foreach ($item in $xml.TestCases.test) {
+        “{0,-60} {1,20} {2,15} {3,15} {4,40}” -f` $item.testName, $item.Platform, $item.Category, $item.Area, $item.Tags
```
Comment (ader1990) [inline Get-LISAv2Statistics.ps1]: make sure when you copy code, to replace the non standard quotation marks with standard quotation marks, and not the literature ones.

“ is different than "

also, remove the back tick from -f`

File: Get-LISAv2Statistics.ps1 [hunk 2]
```diff
@@ -0,0 +1,57 @@
+﻿# Linux on Hyper-V and Azure Test Code, ver. 1.0.0
+# Copyright (c) Microsoft Corporation
+
+# Description: This script displays the LISav2 test case statistics and detail tag.
```
Comment (ader1990) [inline Get-LISAv2Statistics.ps1]: what do you mean by "detail tag"?

File: Get-LISAv2Statistics.ps1 [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+            } else {
+                $tags.Add($single_tag, 1)
+            }
+
+        }
+    }
+}
+
+# Show the statistics information
+Write-Output ""
```
Comment (ader1990) [inline Get-LISAv2Statistics.ps1]: Write-Output has a pretty bad way of showing characters, appears as return value from a function and does not show on some console (like jenkins).

File: Get-LISAv2Statistics.ps1 [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+Write-Output "===== Test Cases counts per platform ====="
```
Comment (ader1990) [inline Get-LISAv2Statistics.ps1]: Test Cases Number*

File: Get-LISAv2Statistics.ps1 (hunk 1 above)
Comment (juhlee-microsoft) [reply Get-LISAv2Statistics.ps1]: Fixed in c583a835aba535a7ccb9ccf1dc847a20e1881fc0

File: Get-LISAv2Statistics.ps1 (hunk 2 above)
Comment (juhlee-microsoft) [reply Get-LISAv2Statistics.ps1]: Changed in c583a835aba535a7ccb9ccf1dc847a20e1881fc0

File: Get-LISAv2Statistics.ps1 (hunk 3 above)
Comment (juhlee-microsoft) [reply Get-LISAv2Statistics.ps1]: This is a change  based on PSScriptAnalyzer result. No plan to run this script by Jenkins yet.

File: Get-LISAv2Statistics.ps1 (hunk 4 above)
Comment (juhlee-microsoft) [reply Get-LISAv2Statistics.ps1]: Fixed in ee06dee7ca4698a2bfa07888ee6c509d081c9b2c

**Pull request merged**